metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1709,19 +1709,17 @@ def _exec_feature_query(self, *, feature_key: str) -> Dict:
 
         feature_query = textwrap.dedent("""\
         MATCH (feat:Feature {key: $feature_key})
-        OPTIONAL MATCH (db:Database)-[:FEATURE]->(feat)
+        OPTIONAL MATCH (db:Database)-[:AVAILABLE_FEATURE]->(feat)
+        OPTIONAL MATCH (fg:Feature_Group)-[:GROUPS]->(feat)
         OPTIONAL MATCH (feat)-[:LAST_UPDATED_AT]->(t:Timestamp)
```

Comment:
not a node, this is an attribute same as `created_timestamp` and called `last_updated_timestamp`.